scripts/sct_download_data.py

```diff
@@ -84,92 +86,174 @@ def main(args=None):
+        ],
+        "binaries_centos": [
+            "https://osf.io/97ybd/?action=download",
+            "https://www.neuro.polymtl.ca/_media/downloads/sct/20181204_sct_binaries_linux_centos6.tar.gz",
+        ],
+        "binaries_osx": [
+            "https://osf.io/zjv4c/?action=download",
+            "https://www.neuro.polymtl.ca/_media/downloads/sct/20181204_sct_binaries_osx.tar.gz",
+        ],
+        "course_hawaii17": "https://osf.io/6exht/?action=download",
+        "course_paris18": [
+            "https://osf.io/9bmn5/?action=download",
+            "https://www.neuro.polymtl.ca/_media/downloads/sct/20180612_sct_course-paris18.zip",
+        ],
+        "course_london19": [
+            "https://osf.io/4q3u7/?action=download",
+            "https://www.neuro.polymtl.ca/_media/downloads/sct/20190121_sct_course-london19.zip",
+        ],
+        "sct_course-beijing19": [
+            "https://osf.io/ef4xz/?action=download",
+            "https://www.neuro.polymtl.ca/_media/downloads/sct/20190802_sct_course-beijing19.zip",
+        ],
+        "deepseg_gm_models": [
+            "https://osf.io/b9y4x/?action=download",
+            "https://www.neuro.polymtl.ca/_media/downloads/sct/20180205_deepseg_gm_models.zip",
+        ],
+        "deepseg_sc_models": [
+            "https://osf.io/avf97/?action=download",
+            "https://www.neuro.polymtl.ca/_media/downloads/sct/20180610_deepseg_sc_models.zip",
+        ],
+        "deepseg_lesion_models": [
+            "https://osf.io/eg7v9/?action=download",
+            "https://www.neuro.polymtl.ca/_media/downloads/sct/20180613_deepseg_lesion_models.zip",
+        ],
+        "c2c3_disc_models": [
+            "https://osf.io/t97ap/?action=download",
+            "https://www.neuro.polymtl.ca/_media/downloads/sct/20190117_c2c3_disc_models.zip",
+        ],
     }
 
     # Get parser info
     parser = get_parser()
     arguments = parser.parse(args)
-    data_name = arguments['-d']
-    verbose = int(arguments.get('-v'))
+    data_name = arguments["-d"]
+    verbose = int(arguments.get("-v"))
     sct.init_sct(log_level=verbose, update=True)  # Update log level
-    dest_folder = arguments.get('-o', os.path.abspath(os.curdir))
+    dest_folder = arguments.get("-o", os.path.abspath(os.curdir))
+    dest_tmp_folder = sct.tmp_create()
+    print("Temp folder: " + dest_tmp_folder)
 
     # Download data
     url = dict_url[data_name]
     tmp_file = download_data(url, verbose)
 
-    # Check if folder already exists
-    sct.printv('\nCheck if folder already exists...', verbose)
-    if os.path.isdir(data_name):
-        sct.printv('WARNING: Folder ' + data_name + ' already exists. Removing it...', 1, 'warning')
-        sct.rmtree(data_name)
+    # Extracting the binaries doesn't require removing the previously created directories
+    # since binaries are files, not directories
+    if "binaries" in data_name:
```

Comment:
will break if compressed files don't have the term "binaries" in it. Why not simply:
- check if the uncompressed zip gives you a folder
- check if the folder already exists
- if it does, remove it

Reply:
Yeah, I figured out that this would happen, I'm currently working on a fix. to have this done in a cleaner way.